lib/resources/iis_app.rb

```diff
@@ -0,0 +1,111 @@
+# encoding: utf-8
+# frozen_string_literal: true
+# check for web applications in IIS
+# Usage:
+# describe iis_app('/myapp', 'Website') do
```

Comment:
I think we do not need the usage comments here, since it is part of the inline `examples`